libpod/oci.go

```diff
@@ -327,7 +311,26 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string) (er
 		cmd.ExtraFiles = append(cmd.ExtraFiles, fds...)
 	}
 
-	err = cmd.Start()
+	if selinux.GetEnabled() {
+		// Set the label of the conmon process to be level :s0
+		// This will allow the container processes to talk to fifo-files
+		// passed into the container by conmon
+		plabel, err := selinux.CurrentLabel()
+		if err != nil {
+			childPipe.Close()
+			return errors.Wrapf(err, "Failed to get current SELinux label")
+		}
+
+		c := selinux.NewContext(plabel)
+		if c["level"] != "s0" && c["level"] != "" {
+			c["level"] = "s0"
+			label.SetProcessLabel(c.Get())
```

Comment:
Is it per thread?  Do we need to pin the OS thread?

Reply:
Yes it is per thread, I was thinking the same thing.

Reply:
Do you know what the code is to do that?

Reply:
@rhatdan https://github.com/projectatomic/libpod/blob/master/libpod/oci.go#L203